### ISSUE TITLE
A11y: B2B Components lack descriptive page titles

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5509]
All pages must have a {{<title>}} that describe topic or purpose. Current storefront pages follow this pattern: 
{{<name of page> | Demostore}}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39646**



[created from NEXT-39646, comment 513190]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-39646, comment 516514]
